recipes/zarp/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "zarp" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0defdec6fe8dabf66d40f2f4d16dfc2619f509f11763356908af3be51ca049ab
+
+build:
+  noarch: python
+  entry_points:
+   - zarp = zarp.cli:main
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - addict >=2.4.0
+    - bidict >=0.22.0
+    - email-validator >=1.2.1
+    - jsonref >=0.2
+    - pandas >=1.3.5
+    - pydantic >=1.9.2 
+    - pygments >=2.8.0
+    - python
+    - rich >=12.5.1
+    - pip
+    - pip:
+      - pydantic-collections >=0.3.0
+  run:
+    - addict >=2.4.0
+    - bidict >=0.22.0
+    - email-validator >=1.2.1
+    - jsonref >=0.2
+    - pandas >=1.3.5
+    - pydantic >=1.9.2
+    - pygments >=2.8.0
+    - python
+    - rich >=12.5.1
+    - pip
+    - pip:
+      - pydantic-collections >=0.3.0
+test:
+  imports:
+    - tests
+    - tests.config
+    - zarp
+    - zarp.config
+    - zarp.extensions
+    - zarp.run
+  commands:
+    - zarp --help
+
+
+about:
+  home: The package home page
+  license: Apache License 2.0
+  license_family: APACHE 
+  license_file: 
```

Comment:
Guess you could add a file `LICENSE` with the contents from [here](https://raw.githubusercontent.com/zavolanlab/zarp-cli/dev/LICENSE) and then put `LICENSE` here.

Reply:
I think we can leave that empty. Apache License 2.0 is obvious I think...

Reply:
Looking at Björn's `conda skeleton pypi` output, I think we can just put `LICENSE` here. Probably it will pull from PyPI. I think it's good practice to include the license. We can at least give it a try.